code/modules/hydroponics/grown/towercap.dm

```diff
@@ -54,7 +54,10 @@
 /obj/item/weapon/grown/log/attackby(obj/item/weapon/W, mob/user, params)
 	if(W.sharpness)
 		user.show_message("<span class='notice'>You make [plank_name] out of \the [src]!</span>", 1)
-		var/obj/item/stack/plank = new plank_type(user.loc, 1 + round(seed.potency / 25))
+		var/seed_modifier = 0
+		if(seed)
+			seed_modifier = round(seed.potency / 25)
+		var/obj/item/stack/plank = new plank_type(user.loc, 1 + seed_modifier)
```

Comment:
I didn't change this part, I think it combines them into one stack before placing them on the ground later down in the proc